package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mathjax-node-svg2png",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
On second thought, having a default scaling factor is actually a bug fix -- but it should simply be `data.ex` (default `6`) instead of `38` (which I miscalculated).